tests/components/google/test_calendar.py

```diff
@@ -244,39 +187,7 @@ async def test_offset_in_progress_event(hass, mock_events_list_items):
     }
 
 
-@pytest.mark.skip
```

Comment:
I decided to just remove this given its been disabled for awhile.

Reply:
I've applied these fixes, thanks. I removed any use of "dt_util.dt" which i think is just referencing an import via the dt_util package which does not seem right.